File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    split_array = s.split(" ")
    for word in split_array:
        for k in range(len(word)):
            if (k %2 == 0):
                answer += word[k].upper()
            else:
                answer +=word[k].lower()
        answer+=" "
    return answer[0:-1]

File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    int index = 0;
    string answer;
    for ( int i = 0; i < s.length(); i++){
        if ( s[i] == ' ' ){
             index = 0;
            continue;
        }
        if (index %2 == 0){
            s[i] = toupper(s[i]);
        }
        else s[i] = tolower(s[i]);
        index++;

}
    answer += s;  
     return answer;
}